Homework_1_User_input/Homework_1_User_input/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace Homework_1_User_input
+{
+    class Program
+    {
+       
+        static void Main(string[] args)
+        {
+            for (int i = 0; i<=1;) { 
```

Comment:
If instead of a loop you move the body of this function into a function of its own- you won't need a loop and will simply be able to call the function twice.